src/shared-components/EventStats/css/Loader.module.css

```
@@ -0,0 +1,45 @@
+  border-top: var(--space-4) solid var(--color-white-20);
+  border-right: var(--space-4) solid var(--color-white-20);
+  border-bottom: var(--space-4) solid var(--color-white-20);
+  border-left: var(--space-4) solid var(--color-yellow-500);
+  -webkit-transform: translateZ(var(--space-0));
+  -ms-transform: translateZ(var(--space-0));
+  transform: translateZ(var(--space-0));
+  -webkit-animation: load8 var(--animation-duration-md) infinite linear;
+  animation: load8 var(--animation-duration-md) infinite linear;
+}
+
+@-webkit-keyframes load8 {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed at-rule "@-webkit-keyframes" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)